src/Components/Components.sln

```diff
@@ -236,6 +232,12 @@ Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "Microsoft.AspNetCore.Compon
 EndProject
 Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "Microsoft.AspNetCore.Components.Forms.Tests", "Forms\test\Microsoft.AspNetCore.Components.Forms.Tests.csproj", "{173D84A3-0F37-480F-AC0F-7E2DBBE32B28}"
 EndProject
+Project("{2150E333-8FDC-42A3-9474-1A3956D46DE8}") = "Samples", "Samples", "{46E4300C-5726-4108-B9A2-18BB94EB26ED}"
+EndProject
+Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "BlazorServer.Sample", "Samples\BlazorServer.Sample\BlazorServer.Sample.csproj", "{2D7B741B-FD32-4BA6-9C98-CCC5B424B005}"
+EndProject
+Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "Microsoft.AspNetCore.HttpsPolicy", "..\Middleware\HttpsPolicy\src\Microsoft.AspNetCore.HttpsPolicy.csproj", "{4664276D-606A-4BB3-873A-9EE84FB22877}"
+EndProject
```

Comment:
Why add the `HttpsPolicy` project to this sln?

Reply:
The sample app needs it and I want to keep https enabled in the sample to reflect the template as close as possible

Reply:
Ah yes. I've been working from the `.slnf` for so long I forgot the `.sln` has to include all the shared-framework-type dependencies.